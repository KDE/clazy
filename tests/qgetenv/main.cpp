#include <QtCore/qglobal.h>
#include <QtCore/qbytearray.h>

int test()
{

    qgetenv("Foo").isEmpty();
    bool b = qgetenv("Foo").isNull();
    QByteArray ba = qgetenv("Foo");
    int ignoreOK = qgetenv("Foo").toInt();
    int checkOK = qgetenv("Foo").toInt(&b);
    int dontfix = qgetenv("Foo").toInt(&b, 16);
    return 0;
}
