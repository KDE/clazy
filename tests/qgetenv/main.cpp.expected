qgetenv/main.cpp:7:5: warning: qgetenv().isEmpty() allocates. Use qEnvironmentVariableIsEmpty() instead [-Wclazy-qgetenv]
qgetenv/main.cpp:8:14: warning: qgetenv().isNull() allocates. Use qEnvironmentVariableIsSet() instead [-Wclazy-qgetenv]
qgetenv/main.cpp:10:20: warning: qgetenv().toInt() is slow. Use qEnvironmentVariableIntValue() instead [-Wclazy-qgetenv]
qgetenv/main.cpp:11:19: warning: qgetenv().toInt() is slow. Use qEnvironmentVariableIntValue() instead [-Wclazy-qgetenv]
